e2e/test/iothub/device/MessageSendFaultInjectionTests.cs

```diff
@@ -237,46 +236,28 @@ public async Task Message_ThrottledConnectionLongTimeNoRecovery_AmqpWs()
         [DoNotParallelize]
         public async Task Message_QuotaExceededRecovery_Amqp()
         {
-            // act
-            Func<Task> act = async () =>
-            {
-                await SendMessageRecoveryAsync(
-                        new IotHubClientAmqpSettings(),
-                        FaultInjectionConstants.FaultType_QuotaExceeded,
-                        FaultInjectionConstants.FaultCloseReason_Boom)
+            await SendMessageRecoveryAsync(
+                    new IotHubClientAmqpSettings(),
+                    FaultInjectionConstants.FaultType_QuotaExceeded,
+                    FaultInjectionConstants.FaultCloseReason_Boom)
                 .ConfigureAwait(false);
-            };
-
-            // assert
-            var error = await act.Should().ThrowAsync<IotHubClientException>();
-            error.And.ErrorCode.Should().Be(IotHubClientErrorCode.DeviceMaximumQueueDepthExceeded);
-            error.And.IsTransient.Should().BeFalse();
         }
 
         [TestMethod]
         [Timeout(LongRunningTestTimeoutMilliseconds)]
         [DoNotParallelize]
         public async Task Message_QuotaExceededRecovery_AmqpWs()
         {
-            // act
-            Func<Task> act = async () =>
-            {
-                await SendMessageRecoveryAsync(
-                        new IotHubClientAmqpSettings(IotHubClientTransportProtocol.WebSocket),
-                        FaultInjectionConstants.FaultType_QuotaExceeded,
-                        FaultInjectionConstants.FaultCloseReason_Boom)
+            await SendMessageRecoveryAsync(
+                    new IotHubClientAmqpSettings(IotHubClientTransportProtocol.WebSocket),
+                    FaultInjectionConstants.FaultType_QuotaExceeded,
+                    FaultInjectionConstants.FaultCloseReason_Boom)
                 .ConfigureAwait(false);
-            };
-
-            // assert
-            var error = await act.Should().ThrowAsync<IotHubClientException>();
-            error.And.ErrorCode.Should().Be(IotHubClientErrorCode.DeviceMaximumQueueDepthExceeded);
-            error.And.IsTransient.Should().BeFalse();
         }
 
         [TestMethod]
         [Timeout(TestTimeoutMilliseconds)]
-        public async Task Message_AuthenticationRecovery_Amqp()
+        public async Task Message_AuthenticationNoRecovery_Amqp()
```

Comment:
`Unauthorized` error code is intransient, so the method call won't succeed or be retried. Correct the test name here.